README.md

```diff
@@ -16,29 +16,33 @@ This is the __benchmarking platform__ for <b>I</b>terative <b>O</b>ptimization <
 
 <b>IOHexperimenter</b> is <i>built on</i>:
 
-* `C++` (tested on `gcc 5.4.0`)
-* `boost.filesystem` library for logging files.
+* `C++`
 
 <b>IOHexperimenter</b> is available for:
 
 * `C++`
 * `R` package [https://github.com/IOHprofiler/IOHexperimenter/tree/R](https://github.com/IOHprofiler/IOHexperimenter/tree/R)
-* `Python` interface (soon to come)
+* `Python` interface
 * `Java` interface (soon to come)
 
 ## Using IOHexperimenter
 
 ### Running Experiments
 
-The __IOHexperimenter__ has been built on `C++` and tested on complier `gcc 5.4.0`. To use the logging of `csv` output files, `boost.filesystem` library is required (To install boost library, please visit [https://www.boost.org](https://www.boost.org)).
+The __IOHexperimenter__ has been built on `C++`.
 
 #### Using IOHexperimenter in C++
 
 If you are using the tool for the first time, please download or clone this branch and run `make` at the root directory of the project. After running `make` to compile,
-* object files will be generated in `build/c/obj`
-* three exectuable files will be generated in `build/c/bin`
+* source files will be generated in `build/Cpp/IOH/`
+* A shared library `libIOH.so` will be generated in `build/Cpp/`
+* three exectuable files will be generated in `build/Cpp/bin/`
+
+To set the working environment with __IOH__, you can copy `libIOH.so` files to the library path `/usr/lib/` or `/usr/local/lib/` and copy the folder of header filers `/IOH/` to the include path `/usr/include/` or `/usr/local/include/`. Otherwise. please link the corresponding path to the `LD_LIBRARY_PATY` and the `CPLUS_INCLUDE_PATH`. Afterwards, you can compile your project as follow:
```

Comment:
please link the corresponding path to the `LD_LIBRARY_PATY` --> please add the corresponding path to the `LD_LIBRARY_PATH`, e.g., echo "export LD_LIBRARY_PATH=/usr/local/include:$LD_LIBRARY_PATH" >> ~/.bashrc